app/templating/summary/group.py

```diff
@@ -10,7 +12,9 @@ def __init__(self, group_schema, path, answer_store, metadata, schema):
         self.title = group_schema.get('title')
 
         self.blocks = self._build_blocks(group_schema, path, answer_store, metadata, schema)
-        self.placeholder_renderer = PlaceholderRenderer(answer_store=answer_store, metadata=metadata)
+        self.placeholder_renderer = PlaceholderRenderer(locale=Locale('en', 'GB'),
```

Comment:
This locale would also need to be dynamic.

Reply:
Summaries are not currently used in Census, so we can leave this defaulted to `en` for now and address this when we re-write summaries

Reply:
Flask context is available here anyway?. But ya sure.

Reply:
But re comments yesterday in "planning". We want to limit how much we use Flask outside of the views so we can switch away easier.